lighthouse-core/test/test-utils.js

```diff
@@ -78,6 +78,63 @@ function getProtoRoundTrip() {
   };
 }
 
+/**
+ * @typedef PartialScriptCoverage
+ * @property {string} url
+ * @property {Array<{ranges: LH.Crdp.Profiler.CoverageRange[]}>} functions
+ */
+
+/**
+ * @param {string} name
+ * @return {{map: LH.Artifacts.RawSourceMap, content: string, usage?: PartialScriptCoverage}}
```

Comment:
if the usage type is always what's given below, augment with dummy `functionName`/`scriptId`/`isBlockCoverage` and make it a real `ScriptCoverage` instead of this `PartialScriptCoverage`?